frontend/src/components/contentDisplay/H5PEditor.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <h5p-editor ref="editor" :content-id="contentId" />
+</template>
```

Comment:
We need a custom save-button, the editor doesn't provide that - could also implement this on the h5p branch

Reply:
Would suggest that, since than it can be made specific for the editor and how we need it